### PR TITLE
Update gitlab.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**
Updated steps and screenshots

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 1]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
 
 Save the generated token to use later.
 
-<img src="https://assets.postman.com/postman-docs/gitlab_create_gray2.png" alt="gitlab create token">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-create-token.jpg)
 
-<img src="https://assets.postman.com/postman-docs/gitlab_token2_gray2.png" alt="gitlab token">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-save-token.jpg)
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:30]: Let's update alt text for these screenshots as well

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 2]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
 ## Configuring a backup for Postman Collections in GitLab
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select GitLab from the list of third party integrations.
+From the [Integrations search page](https://go.postman.co/integrations/browse?category=all) search and select Gitlab.
 
-  <img src="https://assets.postman.com/postman-docs/integrations-gitlab1_gray2.png" alt="select gitlab integration">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-search-all-bb.jpg)
 
-1. Click **View Details** to see information about GitLab and how it can back up your Postman Collections to your GitLab projects or self-hosted domains.
+You can select `View` or `View All`to see all integrations created by your team.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:38]: Let's bold buttons vs using inline code formatting

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 3]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
 
-    **Note:** You can also click the **Configured Integrations** tab to set up other integrations, view available integrations for Gitlab, or view all integrations.
-
-    <img src="https://assets.postman.com/postman-docs/Gitlab_Main_Backup_Page_gray2.png" alt="select gitlab add">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-add-integration.jpg)
 
     The above screen provides you the following two options:
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:42]: This is showing up as oddly formatted on my end—can we remove indents and see if that helps?

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 4]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
+Select **Add Integration** to enter your Gitlab token to start the integration.
 
-<img src="https://assets.postman.com/postman-docs/GitLab_Add1_gray2.png" alt="select gitlab add">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-backup-collection-authenticate.jpg)
 
-1. Enter your GitLab Personal Access Token and click **Proceed**.
+Enter your GitLab Personal Access Token and select `Authenticate and Proceed`.
 
-1. In the **Backup your Postman Collections** page, choose an existing Postman Collection and your GitLab Project,  enter a filename for your backup.
+Choose an existing Postman Collection and your GitLab Project, and enter a filename for your backup. Select `Add Integration` to save your integration configuration.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:55]: Can we add a screenshot here?

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 5]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
 
-1. Click the **Add Integration** button.
 Your Collection is pushed to your GitLab project under the filename that you specified and saved as a single JSON file.
 
-    Now every change that is saved to your Postman Collection automatically commits changes to your GitLab project in real time. Your Collections and code can exist in the same repository.
+  Now every change that is saved to your Postman Collection automatically commits changes to your GitLab project in real time. Your Collections and code can exist in the same repository.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:59]: Second "Collections" should be lowercase, see [here](https://postmanlabs.atlassian.net/l/c/cHHEaJVG)

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 6]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
+### Static IP Support
 
-    **Static IP Support**
+  If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable collection backups to GitLab on custom domains.
 
-    If your network is behind a firewall that requires whitelisted IP addresses, you will need to use a static IP address to enable collection backups to GitLab on custom domains.
+  Contact your IT team to whitelist the following static IP in your firewall to enable collection backups GitLab:
 
-    Contact your IT team to whitelist the following static IP in your firewall to enable collection backups GitLab:
+    * US East: `3.212.102.200`
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:79]: This one is showing up as a code block — might be indenting?

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 7]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
 
-      * US East: `3.212.102.200`
+  Once you whitelist this IP address, calls for this integration will be able to connect to your network and allow the integration to work as expected.
 
-    Once you whitelist this IP address, calls for this integration will be able to connect to your network and allow the integration to work as expected.
+In the **Backup your Postman Collections** page, choose an existing Postman Collection and your GitLab Project, enter a filename for your backup.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:83]: "enter a filename" -> "then enter a filename"

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 8]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
 
-1. In the **Backup your Postman Collections** page, choose an existing Postman Collection and your GitLab Project, enter a filename for your backup.
+Select **Add Integration**.
 
-1. Click **Add Integration**.
-Your Collection is pushed to your GitLab project under the directory that you specified and saved as a single JSON file. This is illustrated in the following screen:
+Your Collection is pushed to your GitLab project under the directory that you specified and saved as a single JSON file.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:87]: Lowercase collection here

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 9]
```diff
@@ -102,7 +98,7 @@ In "Advanced Options" you can enter a custom directory name or leave "Postman Co
 
 <img src="https://assets.postman.com/postman-docs/integrations-gitlab-advOptions1_gray2.png" alt="gitlab advanced options">
 
-Click Add Integrations.
+Select `Add Integrations`.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/gitlab.md:101]: Bold vs inline code

**Review by claireriley — CHANGES_REQUESTED** [review]
Hey @jerry-li-dev — few notes and we can get this pushed shortly!

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 10]
```diff
@@ -25,70 +25,66 @@ If you don’t already have a Personal Access Token from GitLab, [generate a new
...
-<img src="https://assets.postman.com/postman-docs/gitlab_create_gray2.png" alt="gitlab create token">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-create-token.jpg)
 
-<img src="https://assets.postman.com/postman-docs/gitlab_token2_gray2.png" alt="gitlab token">
+![gitlab dashboard](https://assets.postman.com/postman-docs/gitlab-save-token.jpg)
 
 ## Configuring a backup for Postman Collections in GitLab
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select GitLab from the list of third party integrations.
+From the [Integrations search page](https://go.postman.co/integrations/browse?category=all) search and select Gitlab.
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/available-integrations/gitlab.md:34]: Are we using Integration search page as the nomenclature everywhere? I think we should stick to using "Browse Integration" page just to maintain consistency on the UI and docs

**Review by shashankawasthi88 — COMMENTED** [review]
Added comments

File: src/pages/docs/integrations/available-integrations/gitlab.md (hunk 10 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/gitlab.md:34]: Changed to "Browse Integration."

**Comment by jkonrath-postman** [comment]
(This is now PR #3127)

**Pull request closed without merging**